### ISSUE TITLE
ASV benchmarks in bad state and useless

### ISSUE THREAD

**Issue by fmaussion** [OGGM/oggm#1216]
https://cluster.klima.uni-bremen.de/~github/asv/#/ has not been updated since summer 2020

some of the benchmarks now download data from server, etc.

Pretty bad state...

**Comment by TimoRoth** [comment]
You told me quite a while ago to turn them off, hence the lack of updates.

**Comment by fmaussion** [comment]
Yeah no worries, it was more a reminder for me, about all the things that needs to be done...
